src/main/kotlin/de/bund/digitalservice/useid/config/SecurityHeadersFilter.kt

```diff
@@ -24,15 +28,17 @@ class SecurityHeadersFilter(
 
         if (!pathIsValidWidgetPages) return filterChain.doFilter(request, response)
 
-        val hostName: String? = request.getParameter("hostname")
-        val hostNameIsAllowed = hostName?.let { contentSecurityPolicyProperties.domainIsAllowed(it) }
+        val host = request.getParameter("hostname") ?: ""
+        val tenant = tenantProperties.findByAllowedHost(host)
```

Comment:
Why do the resolving again which is already done by the `ResolveTenantFilter?`. Make sure this filter runs after the resolving has happened and read the tenant from the request.